editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -300,21 +301,79 @@ export const useDelayedStrategyCursor = () => {
   return useDelayedStrategy<CSSCursor | null>(selector)
 }
 
+const notResizableControls: ControlWithKey = {
```

Comment:
I wonder if this could be made more "ceremonial" by turning into the official array of "NonApplicableControls" and become an array with one element

Reply:
It was an array at one point, but I suspect because any other cases will also be similarly specific that they'll need to be separate instances.